### PR TITLE
Fixed a couple of typos

### PR DESCRIPTION
**Pull request by hanskokx**
Fixed a couple of typos in the blog section.  There were some misspelled words, which have been corrected.

Signed-off-by: Hans Kokx <hans.d.kokx@gmail.com>

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -64,7 +64,7 @@ <h1> Step 1: Introductions </h1>
 
   <div class="py-5">
   <h1> Step 2: Starter Project </h1>
-  <p>A good way to kick things off with your team is a fun introductionary project. Our team
+  <p>A good way to kick things off with your team is a fun introductory project. Our team
```
Comment (Tragikomedia) [inline blog.html:67]: An interesting change.

File: blog.html [hunk 2]
```diff
@@ -64,7 +64,7 @@ <h1> Step 1: Introductions </h1>
 
   <div class="py-5">
   <h1> Step 2: Starter Project </h1>
-  <p>A good way to kick things off with your team is a fun introductionary project. Our team
```
Comment (Rift3000) [inline blog.html:67]: My engrish is superb. I do not accept this request!

File: blog.html [hunk 3]
```diff
@@ -81,7 +81,7 @@ <h1> Step 3: Check ups </h1>
   <div class="py-5">
   <h1> Step 4: Weekly Reviews </h1>
   <p> Its important to review how everyone is doing at the weekly meetings. 
-   This provides a good oppurtunity to see how well things are going.</p>
+   This provides a good opportunity to see how well things are going.</p>
```
Comment (Tragikomedia) [inline blog.html:84]: Seems sufficient.

**Review by Tragikomedia — APPROVED** [review]

File: blog.html [hunk 4]
```diff
@@ -81,7 +81,7 @@ <h1> Step 3: Check ups </h1>
   <div class="py-5">
   <h1> Step 4: Weekly Reviews </h1>
   <p> Its important to review how everyone is doing at the weekly meetings. 
-   This provides a good oppurtunity to see how well things are going.</p>
```
Comment (Goober0329) [inline blog.html:84]: I like this spelling more.

File: blog.html (hunk 3 above)
Comment (Mantas0726) [inline blog.html:84]: Looking good!

File: blog.html (hunk 2 above)
Comment (hanskokx) [reply blog.html:67]: This is highly suspect...

File: blog.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (hanskokx) [reply blog.html:84]: It certainly does ;P

File: blog.html (hunk 3 above)
Comment (hanskokx) [reply blog.html:84]: Thanks for your review!

File: blog.html (hunk 4 above)
Comment (hanskokx) [reply blog.html:84]: Too bad!

File: blog.html (hunk 1 above)
Comment (hanskokx) [reply blog.html:67]: Why thank you, kind sir.

**Review by Tragikomedia — APPROVED** [review]
Now we're talking

File: blog.html [hunk 5]
```diff
@@ -17,7 +17,7 @@
 <body>
 
 
-<div class="jumbotron jumbotron-fluid jumbotron-image shadow " >
```
Comment (Rift3000) [inline blog.html:20]: My alien overlords know where you live

**Comment by hanskokx** [comment]
+1

**Comment by Tragikomedia** [comment]
+1

**Review by Goober0329 — COMMENTED** [review]
+1

**Comment by yash-1th** [comment]
+1

**Review by Mantas0726 — APPROVED** [review]
+1

**Pull request merged**